sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1651,6 +1675,7 @@ object RapidsConf {
     .doc("Explain why some parts of a query were not placed on a GPU or not. Possible " +
       "values are ALL: print everything, NONE: print nothing, NOT_ON_GPU: print only parts of " +
       "a query that did not go on the GPU")
+    .commonlyUsed()
```

Comment:
Do we have an agreement on which configs are common while which ones are advanced ?

Reply:
Yes, doc [here](https://docs.google.com/spreadsheets/d/1I4YxzdDQYpog_jq7oAQLBwelYlKAofCtgNP_ar8Q4sQ/edit#gid=0).